### ISSUE TITLE
This repo might help solve the Janus problem

### ISSUE THREAD

**Issue by ReRadioActivity** [ashawkey/stable-dreamfusion#186]
**Is your feature request related to a problem? Please describe.**
Multi-face Janus problem

**Describe the solution you'd like**
3D-aware score distillation

**Describe alternatives you've considered**
Maybe use solution presented here? -> https://github.com/KU-CVLAB/3DFuse

**Additional context**
Don't know what license that repo has, maybe you won't be able to use it

**Comment by ashawkey** [comment]
Yes, their coarse depth model and depth-controlled diffusion model can solve the Janus problem, but it requires a lot of changes for this repo to apply these improvements (involving a custom diffusion model, point-e).
